projects/veRocket/index.js

```diff
@@ -1,38 +1,17 @@
 const { get } = require('../helper/http')
-const { toUSDTBalances } = require('../helper/balances');
 
 const baseUrl = "https://api-v1.verocket.com";
 const tvlApy = `${baseUrl}/dex/overall/lp_volume`
-const vetPrice = `${baseUrl}/price/vet`
-
-
-const getVetPrice = async () => {
-  return (await get(vetPrice)).data.usd
-}
 
 async function tvl() {
   balances = 0;
   const response = (await get(tvlApy)).data;
-
-  // get vet price
-  const vetPrice = await getVetPrice();
-
-  // fetch latest TVL from the list of 30 days TVL
-  tvlObjects = response[29].items;
-
-  tvlObjects.forEach(el => {
-    balances += el.eq_vet * vetPrice;
-  });
-
-  return toUSDTBalances(balances)
+  return { 'vechain': +response[response.length - 1].eq_vet}
 }
 
-tvl()
-
 module.exports = {
-  timetravel: true,
```

Comment:
What does the `timetravel` do?

Reply:
it is a flag that lets our scripts know whether protocol data can be refilled, for example, if this had instead used on chain calls, we can send older block number as input and fetch historical tvl, but since we are fetching from server here, no way to get tvl say two months ago.